dnf.spec

```diff
@@ -47,6 +47,8 @@
 %global yum_compat_level full
 %global yum_subpackage_name yum
 %if 0%{?fedora}
+    # Avoid file conflict with yum < 4 in all Fedoras
+    # It can be resolved by pretrans scriptlet but they are not recommended in Fedora
```

Comment:
This is not true. `%pretrans` scriptlets just have to be written in Lua.

Reply:
In the Fedora Packaging Guidelines ( https://docs.fedoraproject.org/en-US/packaging-guidelines/Scriptlets/#pretrans ) is that "%pretrans is best avoided, but if used it MUST (by necessity) be written in Lua". It doesn't sound like they are recommended.

Reply:
You could also just fix the `yum` package accordingly and then have it Conflict with versions below it. Combined with `OrderWithRequires`, you'd be able to avoid the pretrans scriptlet, I think.